hw/ip/hmac/dv/hmac_sim_cfg.hjson

```diff
@@ -45,6 +45,14 @@
   uvm_test: hmac_base_test
   uvm_test_seq: hmac_base_vseq
 
+  // run modes
+  run_modes:[
+      {
+        name: smoke_regression_mode
+        run_opts: ["+smoke_regression=1"]
```

Comment:
You can move this the to test specification below. wrapping it a run_mode is not needed.

Reply:
Sure, I will remove this wrapper. Thanks Sri!